python/cudf/cudf/utils/pandasutils.py

```diff
@@ -0,0 +1,61 @@
+def reorder_dataframe_columns_to_match_pandas(gdf, pdf):
```

Comment:
Lets make this a test utility instead of a general utility. I can't imagine us using this outside of testing.

Reply:
You're right. It's just a test utility. I'll move it.